spec/lib/terraforming/resource/datadog_monitor_spec.rb

```diff
@@ -0,0 +1,197 @@
+require "spec_helper"
+
+module Terraforming
+  module Resource
+    describe DatadogMonitor do
+      let(:client) { instance_double(Dogapi::Client).as_null_object }
+
+      let(:monitors) do
+        [ "200",
+          [
```

Comment:
__[rubocop]__
* Align the elements of an array literal if they span more than one line. [:ref](https://github.com/bbatsov/ruby-style-guide#align-multiline-arrays)

<!-- Comment by [SideCI](https://sideci.com) -->